conda/cli/install.py

```diff
@@ -295,11 +294,12 @@ def install(args, parser, command='install'):
                 raise PackagesNotFoundError(e._formatted_chains, channels_urls)
 
         except (UnsatisfiableError, SystemExit, SpecsConfigurationConflictError) as e:
-            # Quick solve with frozen env failed.  Try again without that.
-            if isinstall and args.update_modifier == NULL:
+            # Quick solve with frozen env or trimmed repodata failed.  Try again without that.
+            if not hasattr(args, 'update_modifier') or args.update_modifier == NULL:
```

Comment:
Removing isinstalled leads to an extra solve when creating an environment with Unsatisfiable specs:

```
$ conda create -n test python=2.7 ipython=7
Collecting package metadata (current_repodata.json): done
Solving environment: failed with current_repodata.json, will retry with next repodata source.
Initial quick solve with frozen env failed.  Unfreezing env and trying again.
Solving environment: failed with current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed
Initial quick solve with frozen env failed.  Unfreezing env and trying again.
Solving environment: failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:



Package python conflicts for:
ipython=7 -> prompt_toolkit[version='>=2.0.0'] -> pygments -> setuptools -> certifi[version='>=2016.09'] -> python[version='>=3.6,<3.7.0a0,>=3.7,<3.8.0a0']
python=2.7
```
Since there is no environment to freeze the initial quick solve with a frozen environment is un-necessary and wasteful

Reply:
yep, this needs a fix.

Reply:
I think 3acf0a72e should fix this